libkodimote/audioplayer.cpp

```diff
@@ -37,3 +37,14 @@ Playlist * AudioPlayer::playlist() const
 {
     return m_playList;
 }
+
+void AudioPlayer::open(const PlaylistItem &item, bool resume)
+{
+    if (resume) {
+        qDebug() << "resume is not supported for AudioPlayer";
+    }
+
+    m_playList->clear();
+    m_playList->addItems(item);
+    playItem(0);
```

Comment:
This shouldn't be 0. It should be the index in the playlist of the item the user pressed on.